angularjs-example/README.md

```diff
@@ -1,20 +1,34 @@
 angularjs-example
-========
-
+=============
 ## Setup
 
-```
-npm install -g bower
-npm install
-bower install
-```
+1. This demo runs with local Selenium. 
```

Comment:
Oh, I understand. You mean this selenium-standalone: https://www.npmjs.com/package/selenium-standalone

Yes, I understand that.